kyo-scheduler/jvm-native/src/main/scala/kyo/scheduler/Scheduler.scala

```diff
@@ -529,15 +529,15 @@ object Scheduler {
     object Config {
         val default: Config = {
             val cores             = Runtime.getRuntime().availableProcessors()
-            val coreWorkers       = Math.max(1, Flag("coreWorkers", cores * 10))
+            val coreWorkers       = Math.max(1, Flag("coreWorkers", cores))
```

Comment:
The scheduler was starting with too many threads and the regulator taking too long to reach the ideal concurrency